lib/web_ui/dev/roll_fallback_fonts.dart

```diff
@@ -531,27 +516,54 @@ const List<String> apiFallbackFonts = <String>[
 /// handling.
 const List<String> splitFallbackFonts = <String>[
   'Noto Color Emoji',
+  'Noto Sans Symbols 2',
+  'Noto Sans Cuneiform',
+  'Noto Sans Duployan',
+  'Noto Sans Egyptian Hieroglyphs',
+  'Noto Sans HK',
+  'Noto Sans JP',
+  'Noto Sans KR',
+  'Noto Sans SC',
+  'Noto Sans TC',
 ];
 
-Future<bool> _checkForLicenseAttribution(
+String getUrlSuffix(Uri fontUri) {
+  final String urlString = fontUri.toString();
+  if (!urlString.startsWith(expectedUrlPrefix)) {
+    throw ToolExit('Unexpected url format received from Google Fonts API: $urlString.');
+  }
+  return urlString.substring(expectedUrlPrefix.length);
+}
+
+Future<String?> _checkForLicenseAttributions(
```

Comment:
document that this method returns a non-null String with the non-conforming license URL if the license fails the check

Reply:
Done